react/features/welcome/components/WelcomePage.native.js

```diff
@@ -169,6 +226,7 @@ class WelcomePage extends AbstractWelcomePage {
     _onFieldFocusChange(focused) {
         if (focused) {
             // Stop placeholder animation.
+            // $FlowExpectedError
```

Comment:
ditto

Reply:
Cannot call this._clearTimeouts because property _clearTimeouts is missing in WelcomePage [1].